softgroup/basis/common-dao/common-dao-impl/src/main/resources/liquibase/db-changelog-1.0.xml

```diff
@@ -60,5 +60,18 @@
             </sql>
         </rollback>
     </changeSet>
+    
+    <changeSet id="contacts-table-creating-4" author="AlexKazmirchuk">
+        <createTable tableName="contacts">
+            <column name="id" type="varchar(255)">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="name" type="varchar(255)"/>
+            <column name="phone" type="varchar(255)"/>
+            <column name="profile_id" type="varchar(255)">
```

Comment:
@Dominys fixed (I know I should not change change sets ,but when I will have all tables I will add new change sets instead changing old)